packages/core/src/App/app.jsx

```diff
@@ -116,7 +132,7 @@ const AppWithoutTranslation = ({ root_store }) => {
                                     <P2PSettingsProvider>
                                         <TranslationProvider defaultLang={language} i18nInstance={i18nInstance}>
                                             {/* This is required as translation provider uses suspense to reload language */}
-                                            <React.Suspense fallback={<Loading />}>
+                                            <React.Suspense fallback={getLoader()}>
```

Comment:
this new line with `<Loading />` has been added recently by another team. we have to add this fix to avoid showing a regular Loader before our DTrader V2 Skeleton loader.